macos/package_files/3.10.0/preinstall.sh

```diff
@@ -64,8 +75,8 @@ if [[ ${new_uid} != ${new_gid} ]]
 fi
 
 # Stops the agent before upgrading it
-if [ -f /Library/Ossec/bin/ossec-control ]; then
-    /Library/Ossec/bin/ossec-control stop
+if [ -f $DIR/bin/ossec-control ]; then
```

Comment:
Change $DIR to ${DIR}.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/3a2266309751f5e616ee1e44cf19e96350617fa9 and https://github.com/wazuh/wazuh-packages/pull/221/commits/1cb5ebf17dba1bb08ef9d4b8eee71a706945d8ac